src/Histogram.js

```diff
@@ -22,6 +22,11 @@ var OncoHistogram;
 OncoHistogram = function (params, s, rotated) {
     var _self = this;
 
+    _self.lineWidthOffset = 10;
+    _self.lineHeightOffset = 5;
```

Comment:
sure! they were just hardcoded before. What would they be called? `histogramBorderPadding`?